# Returns a list with all the available states/positions
---
string[] ret
